### PR TITLE
modify parse.py, add parse vol,issue,has_abstract

### PR DESCRIPTION
**Pull request by wanweiwei007**
I test the updated version with dump_xml_damage_pubmonth.xml and it works to parse "volume", "issue", and "has_abstract"

### CODE REVIEW

File: wos_parser/parse.py [hunk 1]
```diff
@@ -434,6 +434,26 @@ def parse_reference(branch):
     return success, result_dict
 
 
+##  modified to add vol,issue, has_abstract information
+def parse_vol_issue_hasAbs(branch, global_year, path = pubinfo_path):
```
Comment (alexander-belikov) [inline wos_parser/parse.py]: lower case please : parse_vol_issue_hasAbs -> parse_vol_issue_has_abs

File: wos_parser/parse.py [hunk 2]
```diff
@@ -1036,10 +1056,18 @@ def parse_record(pub, global_year):
 
         page_dict = parse_page(pub, page_path)
 
+        #add vol and issue information
+        volissue_dict = parse_vol_issue_hasAbs(pub, pubinfo_path)
```
Comment (alexander-belikov) [inline wos_parser/parse.py]: success, vol_issue_dict = parse_vol_issue_has _abs(pub, pubinfo_path)

naming it volissue_dict is misleading : the returned result is a tuple

Comment (wanweiwei007) [reply wos_parser/parse.py]: rename it. Please check it.

**Review by alexander-belikov — APPROVED** [review]
great! approving

**Pull request merged**